Tasks/quotes.js

```diff
@@ -1,30 +1,17 @@
 // Change double quotation to open or close quotation
 
-const EMPTY = '';
+'use strict';
 
-quotes = function (s) {
-  res = [];
-  open = false;
-  for (c of s) {
-    if (c === '"') {
-      for (i of c) {
-        if (!open) {
-          res.push('«');
-          open = true;
-        } else {
-          res.push('»');
-          open = false;
-        }
-      }
-    } else {
-      if (c !== '"') {
-        for (i of c) {
-          res.push(i);
-        }
-      }
-    }
+const quotes = (string) => {
+  let res = '';
+  let odd = true;
+  for (const symbol of string) {
+    if (symbol === '"') {
+      res += (odd ? '«' : '»');
+      odd = !odd;
+    } else res += symbol;
```

Comment:
If you have block `{}` in positive branch you need to use `{}` on else branch

Reply:
I fixed all in my branch. Here is a link [https://github.com/mari4kaa/Exams/tree/exams](url)